rubicon_ml/client/rubicon.py

```diff
@@ -120,12 +128,18 @@ def get_project(self, name=None, id=None):
             raise ValueError("`name` OR `id` required.")
 
         if name is not None:
-            project = self.repository.get_project(name)
-            project = Project(project, self.config)
+            for repo in self.repositories:
+                try:
+                    project = repo.get_project(name)
+                except Exception as err:
+                    return_err = err
+                else:
+                    project = Project(project, self.config)
+                    return project
         else:
             project = [p for p in self.projects() if p.id == id][0]
-
-        return project
+            return project
+        raise RubiconException(return_err)
```

Comment:
this is similar to the experiment/dataframe/artifact getter issue except this time you only put the try/except in the first block instead of only in the second one. it should have a similar fix